README.md

```diff
@@ -681,6 +681,16 @@ What is it? Resource that represents residence hall information that would conce
 
 `api/housing/admin/apartment/applications` Gets the apartment application info for all applications submitted during the current semester, otherwise returns a Not Found code
 
+##### PUT
+
+`apartment/applications/{applicationID}/submit` Changes the date an application was submitted (changes it from null the first time they submit)
+
```

Comment:
The edit endpoint is also a PUT, which would be `api/housing/apartment/applications/{applicationID}`